Assets/Scripts/Crew.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+using UnityEngine.UI;
+
+public class Crew : MonoBehaviour
+{
+	private string[] maleNames = File.ReadAllLines("./Assets/ImportedAssets/male_names.txt");
```

Comment:
maybe move these to a singleton or something that cant be run twice? If we end up loading a different scene and come back, they will have to reload the files